lib/cocoapods/generator/copy_xcframework_script.rb

```diff
@@ -226,12 +222,7 @@ def install_xcframework_args(xcframework, slices)
         is_framework = xcframework.build_type.framework?
         args << shell_escape(is_framework ? 'framework' : 'library')
         slices.each do |slice|
-          args << if is_framework
-                    shell_escape(slice.path.relative_path_from(root))
-                  else
-                    # We don't want the path to the library binary, we want the dir that contains it
-                    shell_escape(slice.path.dirname.relative_path_from(root))
-                  end
+          args << shell_escape(slice.path.dirname.relative_path_from(root))
```

Comment:
hah, glad this was already supported in one code path :)

Reply:
I was relieved to find it 😅